chrome/content/zotero/zoteroPane.xul

```diff
@@ -36,7 +36,8 @@
 
 <overlay id="zotero"
 	xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
-	
+
+	<script src="feedSettings.js"/>
```

Comment:
This should be at the bottom of the script list, if anything, but why does this need to be in this file? Isn't it only used in feedSettings.xul?